core/src/main/java/com/sequenceiq/cloudbreak/core/bootstrap/service/container/postgres/PostgresConfigService.java

```diff
@@ -56,10 +64,25 @@ public void decorateServicePillarWithPostgresIfNeeded(Map<String, SaltPillarProp
         Map<String, Object> postgresConfig = new HashMap<>();
         if (dbServerConfigurer.isRemoteDatabaseNeeded(cluster)) {
             postgresConfig.put("configure_remote_db", "true");
+        } else {
+            collectEmbeddedDatabaseConfigs(stack, postgresConfig);
         }
         rdsConfigProviderFactory.getAllSupportedRdsConfigProviders().forEach(provider ->
                 postgresConfig.putAll(provider.createServicePillarConfigMapIfNeeded(stack, cluster)));
         return postgresConfig;
     }
 
+    private void collectEmbeddedDatabaseConfigs(Stack stack, Map<String, Object> postgresConfig) {
+        String databaseDirectory = "/var/lib/pgsql/data";
+        boolean databaseDirectoryOnAttachedDisk = false;
+        if (entitlementService.embeddedDatabaseOnAttachedDiskEnabled(INTERNAL_ACTOR_CRN, ThreadBasedUserCrnProvider.getAccountId())) {
```

Comment:
to reflect on this: https://github.com/hortonworks/cloudbreak/pull/9518/files#r530931639
I think this condition should be false on YCloud and only enable this for real cloud providers

Reply:
Embedded database on attached disk will only be enabled next to the entitlement, if there are attached disks.